### ISSUE TITLE
Android: Can't detect PDF files in openFilePicker()

### ISSUE THREAD

**Issue by paul-castro** [jibon57/nativescript-mediafilepicker#120]
**Platform:** Android (emulator and device)

**Issue:** No PDF files detected on using openFilePicker(), only image files

**Code:**
  
    pickFile(type) {
        let extensions = [];
        let that = this;
 
      if (isAndroid) {
            extensions = ['pdf'];
        } else {
            extensions = [kUTTypePDF, kUTTypeText];
        }
        
        let options: FilePickerOptions = {
            android: {
                extensions: extensions,
                maxNumberFiles: 1
            },
            ios: {
                extensions: extensions,
                multipleSelection: false
            }
        };
        
        let mediafilepicker = new Mediafilepicker(); 
        mediafilepicker.openFilePicker(options);
    }

**Comment by MohammedBashiru** [comment]
Yes. I have same issue but It seems to be an issue with android 10.
Its been reported on the Library's repo
https://github.com/fishwjy/MultiType-FilePicker/issues/63

**Comment by nituwda** [comment]
are there solutions for this kind of matter?

**Comment by csimpi** [comment]
same here, still
